src/pages/AuthPage/Auth.scss

```diff
@@ -0,0 +1,138 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+
+// Backgrounds
+$page-background:  url(../../images/bg.jpg) center/cover no-repeat;
+$auth-form-button-background: lightsalmon;
+
+// Fonts
+$auth-form-font-family: Poppins;
+
+//Colors
+$auth-form-title-color: #333333;
+$auth-form-register-link-color: #666666;
+$auth-form-register-link-hover-color: lightsalmon;
+
+
+.limiter {
+    width: 100%;
+    margin: 0 auto;
+}
+
+.auth-form-container {
+    background: $page-background;
```

Comment:
Search for information how to write css better. I mean you need to have some rules about order of styles
For example:
first you define position, display and so on
position: absolute;
top: 1px;
display: flex;
flex-direction:...
then you define box sizes
width: 600px;
height:
then block styles
border
background
...
then fonts styles
font-size: 16px;
color:
...
then other things
transform:
transition: